controllers/openstackdataplanerole_controller.go

```diff
@@ -138,6 +141,11 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 		instance.Status.Conditions.MarkFalse(dataplanev1beta1.SetupReadyCondition, condition.RequestedReason, condition.SeverityInfo, condition.ReadyInitMessage)
 	}
 
+	err = r.EnsureServices(ctx, helper, instance)
```

Comment:
👍🏽

Reply:
OK, if we're doing it elsewhere we can do it  here too, but sounds weird from the design pov.